src/temp/stories/SearchProject.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import SearchProject from '../Project/SearchProject';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof SearchProject> = {
+  component: SearchProject,
+};
+
+export default meta;
+type Story = StoryObj<typeof SearchProject>;
+
+export const SearchInActive: Story = {
+  args: {
+    activeFilter: false,
+    projectList: [
+      'Natural Regeneration',
+      'Mangroves',
+      'Managed Regeneration',
+      'Other Restoration',
+      'Tree Planting',
+      'Agroforestry',
+      'Urban Restoration',
+      'Conservation',
+    ],
+    searchActive: false,
+  },
+};
+
+export const SearchActive: Story = {
+  args: {
+    activeFilter: false,
+    projectList: [],
+    searchActive: true,
+  },
+};
```

Comment:
Not clear on the name here - why is this `SearchActive`? What are the possible states?

Reply:
I've made some minor changes to it 

1. renamed `SearchActive -> active`
2. removed unnecessary states and kept only one state (`i.e isSearch`)